src/develop/develop.h

```diff
@@ -244,7 +244,7 @@ void dt_dev_configure(dt_develop_t *dev, int wd, int ht);
 void dt_dev_invalidate_from_gui(dt_develop_t *dev);
 
 /*
- * exposure plugin hook, set the white level
+ * exposure plugin hook, set the exposure level and the black level
```

Comment:
Here and next:
`exposure level` is kinda meaningless to me.
Maybe either just `exposure` or `exposure compensation`